src/storage.js

```diff
@@ -0,0 +1,34 @@
+//get config
+require('./config');
+//get redis constant searches redis globally 
+
+const Redis = require('ioredis'); //chnage to ioredis
+//connecting client
+const client = new Redis(process.env.REDIS_URL);
+
+//result from require, displays externally
+module.exports = {
+    addFeed: function(name, url, f) {
```

Comment:
`ioredis` supports Promises, so let's make this async and return a Promise instead.

Reply:
I was playing with it for couple of hours now. Could you show me a an example of how it suppose to work with promises?

Reply:
From the ioredis README.md:

```js
// `exec` also returns a Promise:
var promise = redis
  .pipeline()
  .set("foo", "bar")
  .get("foo")
  .exec();
promise.then(function(result) {
  // result === [[null, 'OK'], [null, 'bar']]
});
```